docs/config/microshift-starter.ks

```diff
@@ -16,6 +16,7 @@ part /boot --fstype=xfs --asprimary --size=800
 part pv.01 --grow
 volgroup rhel pv.01
 logvol / --vgname=rhel --fstype=xfs --size=10240 --name=root
+logvol none --thinpool --vgname=rhel --fstype="None" --size=100 --name=thin
```

Comment:
Can you explain why the 100MB size of the pool?

Reply:
Actually it's much too small. 

My feeling is that it'd be less complicated for users to switch the getting-started workflow to using only thin-vols. This would keep the number of storage classes device classes to 1 each, while also enabling snapshotting/cloning workflows.  

From the users' perspective, the existing storage workflows wouldn't change.
From MicroShift's perspective, the default storage class and lvmd would have to be reconfigured for thin vol provisioning (a relatively trivial change). 

I'll get some feedback on what a preferable thin/thick storage allocation would be from the team.

Reply:
The size is up to the user, right? This is just a demo/starter file?

Reply:
Let's specify a 1024 size in all cases. It should be big enough for the dev usage. 
Users will have their own layouts anyway.

Reply:
Right. I guess my concern is really how this would jive with MicroShift's config-defaulting behavior.  As is, MicroShift's out of the box experience wouldn't provide the snapshot/cloning features without the user setting up a thin-pool, and specifying an lvmd.yaml and storage class. Meaning the demo wouldn't be reflective of an user's unconfigured env.

That also brings up whether the defaulting logic should consider thin-pools in addition to vgs, and which it should prioritize.  Maybe we need to consider a VG/thin-pool naming or tagging scheme instead?